e2e-tests/cypress/tests/support/okta_commands.ts

```diff
@@ -35,29 +75,44 @@ Cypress.Commands.add('oktaCreateUser', (user = {}) => {
                 },
             },
         },
-    }).then((response) => {
+    }).then((response: any) => {
```

Comment:
Any chance we can get rid of this any? Can we type the responses?

Looking on other files, seems like the response for a `task` is by default unknown. Two ways to solve this is either assigning the type (that is not much type safer than casting) (e.g. `cy.task<SomeType>(...).then(...)`) or using guards.

0/5, but would be great to remove these any.

Reply:
Sure!